exes/ethercat/CMakeLists.txt

```diff
@@ -42,6 +42,8 @@ if (BUILD_TESTING)
   add_subdirectory(tests)
 endif ()
 
+configure_file("${CMAKE_SOURCE_DIR}/exes/ethercat/systemd/ethercat@.service" "systemd/ethercat@.service")
```

Comment:
It doesn't override the local file but places it in another directory in the build file, to make this more clear I will add `${CMAKE_BINARY_DIR}/systemd/ethercat@.service` so it will be placed with other systemd files in the build directory.

Will add  `${CMAKE_CURRENT_LIST_DIR}/systemd/ethercat@.service` to remove the absolute path dependancy.